docs/account/ldap-login.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enterprise Appstore and Testing Distribution LDAP Login
+metaTitle: Enterprise Appstore and Testing Distribution LDAP Login
+metaDescription: Enterprise Appstore and Testing Distribution LDAP Login
+sidebar_position: 14
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# LDAP Login
+
```

Comment:
The document page overviews are missing. What is the document page describing? What will happen when someone follows the page?